app/controllers/users_controller.rb

```diff
@@ -1,52 +1,77 @@
 # frozen_string_literal: true
 
 class UsersController < ApplicationController
-  before_action :set_user, only: [:destroy]
-  before_action :require_admin
+  before_action :set_user, only: [:destroy, :edit, :update]
 
   # GET /users
   def index
+    authorize User
     @users = User.all.page(params[:page])
   end
 
   # GET /users/new
   def new
+    authorize User
```

Comment:
maybe use a before_action where required to `authorize User`

Reply:
https://stackoverflow.com/questions/23983576/why-does-before-action-authorize-fail-with-wrong-number-of-arguments suggests that before_actions shouldn't be used with pundit.  It seems kind of copy-paste to me too, but pundit in general seems to encourage scattering the authorize calls in the controller methods